orderer/common/follower/block_puller_test.go

```diff
@@ -32,12 +31,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-//go:generate counterfeiter -o mocks/block_verifier.go --fake-name BlockVerifier . blockVerifier
```

Comment:
When I regenerated the mocks in this folder, this one showed up as an untracked file. I removed this since it isn't used anywhere.